### PR TITLE
[Docs] PR for Backup and restore commands need another cleanup

### PR DESCRIPTION
**Pull request by lizayugabyte**
In addition to the linked ticket, this PR will also close [PLAT-3871](https://yugabyte.atlassian.net/browse/PLAT-3871)

### CODE REVIEW

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md [hunk 1]
```diff
@@ -31,70 +30,66 @@ showAsideToc: true
...
-If you want to back up a single table with its indexes, use the [`create_snapshot`](../../../admin/yb-admin/#create-snapshot) command instead:
+To back up a single table with its indexes, use the [`create_snapshot`](../../../admin/yb-admin/#create-snapshot) command instead, as follows:
 
 ```sh
-./bin/yb-admin create_snapshot my_keyspace my_table
+./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> create_snapshot my_keyspace my_table
 ```
 
-When you run either of these commands, it returns a unique ID for the snapshot:
+When you execute either of the preceding commands, a unique ID for the snapshot is return, as per the following output:
```
Comment (ddhodge) [inline docs/content/preview/manage/backup-restore/snapshots-ycql.md:55]:
```suggestion
When you execute either of the preceding commands, a unique ID for the snapshot is returned, as per the following output:
```

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md [hunk 2]
```diff
@@ -109,49 +104,43 @@ To move a snapshot to external storage, gather all the relevant files from all t
...
+    * *<yb_data_dir>* - a directory where YugabyteDB data is stored. The default value is `~/yugabyte-data`.
+    * *<node_number>* - used when multiple nodes are running on the same server (for testing, QA, and development). The default value is `1`.
+    * *<disk_number>* - used when running YugabyteDB on multiple disks with the `--fs_data_dirs` flag. The default value is `1`.
+    * *<table_id>* - the UUID of the table. You can obtain it from the `http://<yb-master-ip>:7000/tables` URL in the Admin UI.
+    * *<tablet_id>* - each table contains a list of tablets. Each tablet has a `<tablet_id>.snapshots` directory that you need to copy.
+    * *<snapshot_id>* - there is a directory for each snapshot, since you can have multiple completed snapshots on each server.
 
-    {{< note >}}
+    This directory structure is specific to `yb-ctl`, which is a local testing tool. In practice, for each server, you would use the `--fs_data_dirs` flag, which is a comma-separated list of paths for the data. It is recommended to have different paths on separate disks. In the `yb-ctl` example, these are the full paths up to the `disk-x`.
```
Comment (ddhodge) [inline docs/content/preview/manage/backup-restore/snapshots-ycql.md:125]: Does "disk-x" refer to disk-number? It's not in the list

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md [hunk 3]
```diff
@@ -109,49 +104,43 @@ To move a snapshot to external storage, gather all the relevant files from all t
...
 
-  {{< /note >}}
-
-1. If you don't want to keep the in-cluster snapshot, it's now safe to [delete it](#delete-a-snapshot).
+If you don't want to keep the in-cluster snapshot, you can safely [delete it](#delete-a-snapshot).
 
 ## Restore a snapshot from external storage
 
-To restore a snapshot that had been [moved to external storage](#move-a-snapshot-to-an-external-storage), do the following:
+You can restore a snapshot that you have [moved to external storage](#move-a-snapshot-to-an-external-storage), as follows:
```
Comment (ddhodge) [inline docs/content/preview/manage/backup-restore/snapshots-ycql.md:133]:
```suggestion
You can restore a snapshot that you have [moved to external storage](#move-a-snapshot-to-external-storage), as follows:
```

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md [hunk 4]
```diff
@@ -169,36 +158,26 @@ To restore a snapshot that had been [moved to external storage](#move-a-snapshot
     Snapshot         a9442525-c7a2-42c8-8d2e-658060028f0e   a9442525-c7a2-42c8-8d2e-658060028f0e
     ```
 
-1. Copy the tablet snapshots.
+1. Copy the tablet snapshots. Use the tablet mappings to copy the tablet snapshot files from the external location to appropriate location, such as `yb-data/tserver/data/rocksdb/table-<tableid>/tablet-<tabletid>.snapshots`.<br>
```
Comment (ddhodge) [inline docs/content/preview/manage/backup-restore/snapshots-ycql.md:161]:
```suggestion
1. Copy the tablet snapshots. Use the tablet mappings to copy the tablet snapshot files from the external location to an appropriate location, such as `yb-data/tserver/data/rocksdb/table-<tableid>/tablet-<tabletid>.snapshots`.<br>
```

**Review by ddhodge — APPROVED** [review]
Minor issues

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/snapshots-ycql.md:55]: Fixed

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/snapshots-ycql.md:125]: It's disk-number. Fixed

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/snapshots-ycql.md:133]: Fixed

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/snapshots-ycql.md:161]: Fixed

**Pull request merged**